### PR TITLE
Stethoscopes now inform the user if the target can be defibrillated

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
tweak: Stethoscopes now inform the user if the target can be defibrillated; the user will hear a "faint, fluttery pulse."
/:cl:

A friend of mine gave me this idea. It gives stethoscopes an actual use beyond safe cracking (given that their only other use is invalidated with a single examine) and allows doctors to quickly see if someone can be defibrillated without using up defibrillator charge on someone that's already too long-dead without having to yank a medical scanner from their belt or backpack.

I'm no medical professional, but I *think* this makes sense from a biological perspective!

### CODE REVIEW

**Comment by jughu** [comment]
>Doctors using anything else then their medscanner

Remove the medscanner check and give every doctor an stethoscope to check it :^)

**Comment by jughu** [comment]
Becauze why would they ever use stethosscope

If the dern medscanner gives exact defib time window

**Comment by alexkar598** [comment]
I always use the stethoscope.....for asthetics

**Comment by FantasticFwoosh** [comment]
PDA medscanner makes all else irrelevant since it is vastly superior, already tells you time to defib & also tells you time of death when you screw up for station records, fits comfortably into the doctor labcoat too.

I am supportive of devolution of medical tools, in my time playing i did make a dedicated effort to use as many alternative health reading methods as possible rather than cheat with pinpoint accuracy electronics.

**Comment by QualityVan** [comment]
I don't think this is necessary and there's probably some better way to put stethoscopes to work. That being said, this is fine. I don't know whether you intentionally left off checking whether damage is sufficient to prevent defib but it's probably fine without it.

File: code/modules/clothing/neck/_neck.dm [hunk 1]
```diff
@@ -72,6 +72,9 @@
 					if(!(M.failed_last_breath || M.losebreath))
 						lung_strength = "healthy"
 
+			if(M.stat == DEAD && heart && world.time - M.timeofdeath < DEFIB_TIME_LIMIT * 10)
+				heart_strength = "<span class='boldannounce'>a faint, fluttery</span>"
```
Comment (RemieRichards) [inline code/modules/clothing/neck/_neck.dm]: "You faintly hear a faint..."

It's faint is it? Never would have guessed.

**Comment by JJRcop** [comment]
@jughu I think redundancy is ok to have.

File: code/modules/clothing/neck/_neck.dm (hunk 1 above)
Comment (Xhuis) [reply code/modules/clothing/neck/_neck.dm]: If I said that I thought it wasn't amusing, I'd be *feinting* the question.

**Pull request merged**